src/StatusQ/Components/StatusChatListItem.qml

```diff
@@ -207,6 +232,93 @@ Rectangle {
             border.color: color
             visible: statusBadge.value > 0
         }
+    }
+
+    DropArea {
+        id: dropArea
+        width: sensor.held ? 0 : parent.width
+        height: sensor.held ? 0 : parent.height
+        onEntered: {
+            reorderDelay.start()
+        }
 
+        onDropped: {
+            reorder(drag.source.originalOrder, statusChatListItem.DelegateModel.itemsIndex)
+        }
+
+        Timer {
+            id: reorderDelay
+            interval: 100
+            repeat: false
+            onTriggered: {
+                if (dropArea.containsDrag)
+                    visualReorder(dropArea.drag.source.DelegateModel.itemsIndex, statusChatListItem.DelegateModel.itemsIndex)
+            }
+        }
     }
+
+    Loader {
+        id:content
+        active: sensor.held
+        sourceComponent: Item {
+            property var globalPosition: getAbsolutePosition(statusChatListItem)
+            parent: QC.Overlay.overlay
+            width: statusChatListItem.width
+            height: statusChatListItem.height
+            Drag.active: sensor.held
+            Drag.hotSpot.x: width / 2
+            Drag.hotSpot.y: height / 2
+            Drag.source: statusChatListItem
+            Component.onCompleted: {
+                x = globalPosition.x
+                y = globalPosition.y
+            }
+
+            RectangularGlow {
+                anchors.fill: cover
+                color: "#22000000"
+                glowRadius: 8
+                cornerRadius: 8
+            }
+            Rectangle {
+                id: cover
+                anchors.fill: parent
+                color: Theme.palette.baseColor4
+                radius: 8
+
+                Row {
+                    spacing: 6
+                    anchors.left: parent.left
+                    anchors.leftMargin: 8
+                    anchors.verticalCenter: parent.verticalCenter
+                    Loader {
+                        id: iconLoader
+                        anchors.verticalCenter: parent.verticalCenter
+                        sourceComponent: !!statusChatListItem.image.source.toString() ?
+                                             statusRoundedImageCmp : statusLetterIdenticonCmp
+                    }
+
+                    StatusBaseText {
+                        anchors.verticalCenter: parent.verticalCenter
+                        width: cover.width - iconLoader.item.width - 22
+                        text: chatName.text
+                        elide: Text.ElideRight
+                    }
+                }
+            }
```

Comment:
This is problematic. By rebuilding the item component this way, we're losing a lot of its properties. The result is that e.g. the font weight changes when it's dragged, or that we lose various badges and icons.

Can we make sure that, when a chat list item is dragged, it doesn't lose its properties?